packages/fetchai/connections/soef/connection.py

```diff
@@ -165,6 +165,7 @@ class SOEFChannel:
     DEFAULT_PERSONALITY_PIECES = ["architecture,agentframework"]
 
     PING_PERIOD = 30 * 60  # 30 minutes
+    FIND_AROUND_ME_REQUEST_DELAY = 2  # seconds
```

Comment:
1 second is enough

Reply:
checked with tests, some times 1 and even 1.5 second is not enough